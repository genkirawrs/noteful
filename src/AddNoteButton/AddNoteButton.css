.add-note-btn {
    text-align:center;
    width:150px;
    display:inline-block;
    border:1px solid black;
    padding:10px;
    font-size:16px;
}
