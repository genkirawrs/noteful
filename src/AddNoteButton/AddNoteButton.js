import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import './AddNoteButton.css';

class AddNoteButton extends Component {
  render(){
    return(
	<Link className='add-note-btn' to={`/addNote/${this.props.folderId}`}>Add Note</Link>
    );
  }

}

export default AddNoteButton;
