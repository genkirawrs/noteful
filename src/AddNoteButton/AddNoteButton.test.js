import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter,Link } from 'react-router-dom';
import renderer from 'react-test-renderer';
import AddNoteButton from './AddNoteButton';

it('add note button renders without crashing', () => {
const tree = renderer.create(
    <BrowserRouter>
      <AddNoteButton/>
    </BrowserRouter>
  )
  .toJSON();
 expect(tree).toMatchSnapshot();
});
