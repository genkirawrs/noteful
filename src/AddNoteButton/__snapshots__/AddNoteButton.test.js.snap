// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add note button renders without crashing 1`] = `
<a
  className="add-note-btn"
  href="/addNote/none"
  onClick={[Function]}
>
  Add Note
</a>
`;
